contrib/storage-elasticsearch/pom.xml

```diff
@@ -88,36 +97,17 @@
         <artifactId>maven-surefire-plugin</artifactId>
         <configuration>
           <forkCount combine.self="override">1</forkCount>
+          <includes>
+            <include>**/TestElasticsearchSuit.class</include>
+          </includes>
+          <excludes>
+            <exclude>**/ElasticComplexTypesTest.java</exclude>
```

Comment:
Just checking, do we exclude these tests because they rely on testcontainers?

Reply:
It is excluded because now tests will be running as a test suite.